docs/src/index.md

```diff
@@ -1,91 +1,75 @@
 # KernelAbstractions
 
-`KernelAbstractions.jl` is a package that allows you to write GPU-like kernels that
-target different execution backends. It is intended to be a minimal, and performant
-library that explores ways to best write heterogenous code.
+[`KernelAbstractions.jl`](https://github.com/JuliaGPU/KernelAbstractions.jl) (KA) is
+a package that allows you to write GPU-like kernels targetting different
+execution backends. KA intends to be a minimal and
+performant
+library that explores ways to write heterogeneous code. Although parts of
+the package are still experimental, it has been used successfully as part of the
+[Exascale Computing Project](https://www.exascaleproject.org/) to run Julia code
+on pre-[Frontier](https://www.olcf.ornl.gov/frontier/) and
+pre-[Aurora](https://www.alcf.anl.gov/aurora)
+systems. Currently, profiling and debugging require backend-specific calls like, for example, in
+[`CUDA.jl`](https://cuda.juliagpu.org/dev/development/profiling/).
 
 !!! note
-    While `KernelAbstraction.jl` is focused on performance portability, it is GPU-biased
-    and therefore the kernel language has several constructs that are necessary for good
-    performance on the GPU, but may hurt performance on the CPU.
-
-## Quickstart
-
-### Writing your first kernel
-
-Kernel functions have to be marked with the [`@kernel`](@ref). Inside the `@kernel` macro
-you can use the [kernel language](@ref api_kernel_language). As an example the `mul2` kernel
-below will multiply each element of the array `A` by `2`. It uses the [`@index`](@ref) macro
-to obtain the global linear index of the current workitem.
-
+While KernelAbstraction.jl is focused on performance portability, it emulates GPU semantics and therefore the kernel language has several constructs that are necessary for good performance on the GPU, but serve no purpose on the CPU.
+In these cases, we either ignore such statements entirely (such as with `@synchronize`) or swap out the construct for something similar on the CPU (such as using an `MVector`  to replace `@localmem`).
```

Comment:
We don't ignore `@synchronize` we split the inner for Loops at those points

Reply:
So to make this clearer:

```
   I = @index ....
   # codeA
   @synchronize
   # code B
```

Get's lowered to

```
   for idx in workgroup
       I = calculate(idx)
       # codeA
   end
   for idx in workgroup
       I = calculate(idx)
       # codeB
   end
```

Take a look at the `@macroexpand` of `@kernel` to see this pattern.